main.go

```diff
@@ -14,6 +14,8 @@ import (
 
 var version = "unknown"
 
+const applySleepSeconds = 1
```

Comment:
I wonder if this can be an optional flag that we can set depending on the network latency of different IaaSes? Also is 1s a tad too frequent? Can we go to 10s?